### ISSUE TITLE
Can’t make Netatalk 3.1.7 on Debian Wheezy

### ISSUE THREAD

**Issue by ambroisemaupate** [Netatalk/Netatalk#10]
``` shell
  CC     libutil_la-queue.lo
  CC     libutil_la-server_child.lo
  CC     libutil_la-server_ipc.lo
  CC     libutil_la-server_lock.lo
  CC     libutil_la-socket.lo
  CC     libutil_la-strdicasecmp.lo
  CC     libutil_la-unix.lo
unix.c: In function 'ochmod':
unix.c:272:19: error: 'O_IGNORE' undeclared (first use in this function)
unix.c:272:19: note: each undeclared identifier is reported only once for each function it appears in
make[3]: *** [libutil_la-unix.lo] Erreur 1
make[3]: quittant le répertoire « /root/netatalk-code/libatalk/util »
make[2]: *** [all-recursive] Erreur 1
make[2]: quittant le répertoire « /root/netatalk-code/libatalk »
make[1]: *** [all-recursive] Erreur 1
make[1]: quittant le répertoire « /root/netatalk-code »
make: *** [all] Erreur 2
```

Here is my ./configure summary

``` shell
Compilation summary:
    CPPFLAGS       = -I$(top_srcdir)/include -I$(top_builddir)/include -I$(top_srcdir) 
    CFLAGS         = -D_U_="__attribute__((unused))" -g -O2
    LIBS           = -ldl 
    PTHREADS:
        LIBS   = 
        CFLAGS = -pthread
    SSL:
        LIBS   =  -L/usr/lib64 -lcrypto
        CFLAGS =  -I/usr/include/openssl
    LIBGCRYPT:
        LIBS   = -L/lib/x86_64-linux-gnu -lgcrypt
        CFLAGS = 
    BDB:
        LIBS   =  -L/usr/lib64 -ldb-5.1
        CFLAGS = 
    LIBEVENT:
        bundled
    TDB:
        bundled
Configure summary:
    INIT STYLE:
         debian-sysv
    AFP:
         Extended Attributes: ad | sys
         ACL support: no
         Spotlight: no
    CNID:
         backends:  dbd last tdb
    UAMS:
         DHX     ( SHADOW)
         DHX2    ( SHADOW)
         RANDNUM (afppasswd)
         clrtxt  ( SHADOW)
         guest
    Options:
         Zeroconf support:        no
         tcp wrapper support:     no
         quota support:           yes
         admin group support:     yes
         valid shell check:       yes
         cracklib support:        no
         ACL support:             auto
         Kerberos support:        auto
         LDAP support:            no
         AFP stats via dbus:      no
         dtrace probes:           no
    Paths:
         Netatalk lockfile:       /var/lock/netatalk
         init directory:          /etc/init.d
    Documentation:
         Docbook:                 no
```

And my `uname -a`

`Linux edgar 3.16.0-0.bpo.4-amd64 #1 SMP Debian 3.16.7-ckt11-1~bpo70+1 (2015-06-08) x86_64 GNU/Linux`

**Comment by ambroisemaupate** [comment]
This line seems to be responsible 

https://github.com/Netatalk/Netatalk/blob/master/libatalk/util/unix.c#L272

**Comment by hat001** [comment]
Please post to ML or sourceforege.
https://lists.sourceforge.net/lists/listinfo/netatalk-admins
https://lists.sourceforge.net/lists/listinfo/netatalk-devel
http://sourceforge.net/p/netatalk/bugs/

not here.

**Comment by andy-js** [comment]
I think this issue has now been resolved by d48ecb55ac4c916cd78bbaf4de53a04abf6d47fb
